.copilot/config.yml

```
@@ -0,0 +1,4 @@
+repository: public.ecr.aws/data-hub-helpcentre
```

```suggestion
repository: dshc
```

You'll want to update this before merging.